lib/msal-common/docs/Accounts.md

```diff
@@ -0,0 +1,63 @@
+# Accounts in MSAL JS
+
+MSAL JS libraries (`msal-browser` and `msal-node`) support both single account and multiple accounts scenarios in java script applications. An `account` object is standardized across the libraries:
+
+// TODO replace this with typedocs link when available
+
+```javascript
+export type AccountInfo = {
+    homeAccountId: string;
+    environment: string;
+    tenantId: string;
+    username: string;
+};
+```
+
+## Usage
+
+* We provide a public API `getAllAccounts()` which lists all the accounts currently in the cache. An application must choose an account to acquire tokens silently.
+* `msal-browser` provides a public API `getAccountByUsername()` for an application to retrieve a specific account based on `username`
+
+Sample usage is as below:
+
+``` javascript
+
+function handleResponse(resp) {
+    if (resp !== null) {
```

Comment:
might want an example that doesn't have `showWelcomeMessage`

Reply:
removed all `showWelcomeMessage()` calls